types/hapi/v17/test/server/server-info.ts

```diff
@@ -0,0 +1,15 @@
+//  https://github.com/hapijs/hapi/blob/master/API.md#-serverinfo
+import {Server} from "hapi";
+import * as assert from "assert";
+
+const server = new Server({
+    port: 8000,
+});
+server.start();
+
+// check the correct port
+console.log(server.info);
+if (server.info) assert(server.info.port === 8000);
```

Comment:
Do the `assert` checks actually get executed? I thought the tests were mainly to see if it compiles without errors? Not to actually make unit tests. Unit tests seem to be a bit of an overkill for me because the whole code is already unit tested in the "regular" hapijs project. So my assumption is -> if it compiles correctly to JS -> then the "normal" unit tests will have tested the code correctly anyway. That is why `assert`'s are not mandatory :)

Reply:
@BorntraegerMarc Yes, you are right. The most important is compiling. I am executing as well but the assert's is too much. 3d6b184